backend/routers/todoRouters.js

```diff
@@ -0,0 +1,20 @@
+import { Router } from "express";
+import { getAllTodos, addTask, deleteTask, updateTask } from '../controllers/todoControllers.js'
+
+
+const router = new Router();
+
+// Получение всех задач
+router.get('/', getAllTodos);
+
+// Создание новой задачи
+router.post('/',  addTask);
+
+// Удаление задачи
+router.delete('/:id', deleteTask)
+
+// Редактирование задачи
+router.patch('/:id', updateTask)
```

Comment:
Why can't `todoControllers` call the `router.method` by itself? Why do you need to export the operations as functions only to use them in `router`?